subjects/0-shell/job-control/README.md

```diff
@@ -0,0 +1,51 @@
+## job control
+
+### Objectives
+
+You must follow the same [principles](https://public.01-edu.org/subjects/0-shell/) as the first subject.
+
+Job control refers to the ability to selectively stop (suspend) the execution of processes and continue (resume) their execution at a later point.
+
+In `job control`, you will have to implement the following [builtins](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Job-Control-Builtins):
+
+- jobs
+- bg
+- fg
+- kill
+- disown
```

Comment:
To follow BusyBox I would replace `disown` with `nohup` or remove this builtin entirely as it is less important.